web/frontend/src/components/slam.vue

```diff
@@ -228,100 +343,208 @@ const refresh3dMap = () => {
+                  </option>
+                  <option value="5">
+                    Every 5 seconds
+                  </option>
+                  <option value="1">
+                    Every second
+                  </option>
+                </select>
+                <div
+                  class="pointer-events-none absolute inset-y-0 right-0 flex items-center px-2"
+                >
+                  <svg
+                    class="h-4 w-4 stroke-2 text-gray-700"
+                    viewBox="0 0 24 24"
+                    stroke="currentColor"
+                    stroke-linejoin="round"
+                    stroke-linecap="round"
+                    fill="none"
                   >
-                    <svg
-                      class="h-4 w-4 stroke-2 text-gray-700"
-                      viewBox="0 0 24 24"
-                      stroke="currentColor"
-                      stroke-linejoin="round"
-                      stroke-linecap="round"
-                      fill="none"
-                    >
-                      <path d="M18 16L12 22L6 16" />
-                    </svg>
-                  </div>
+                    <path d="M18 16L12 22L6 16" />
+                  </svg>
                 </div>
               </div>
-              <div class="px-2 pt-7">
-                <v-button
-                  v-if="show2d"
-                  icon="refresh"
-                  label="Refresh"
-                  @click="refresh2dMap()"
-                />
+            </div>
+            <div class="px-2 pt-11">
+              <v-button
+                label="Refresh"
+                icon="refresh"
+                @click="refresh2dMap()"
+              />
+            </div>
+          </div>
+          <hr class="my-4 border-t border-gray-400">
+          <div class="flex flex-row">
+            <p class="mb-1 pr-52 font-bold text-gray-800">
+              Ending Position
+            </p>
+            <v-icon
+              name="trash"
+              @click="executeDeleteDestinationMarker()"
+            />
+          </div>
+          <div class="flex flex-row pb-2">
+            <v-input
+              type="number"
+              label="x"
+              incrementor="slider"
+              :value="destinationMarker.x"
+              step="0.1"
+              @input="handleUpdateX($event)"
+            />
+            <v-input
+              class="pl-2"
+              type="number"
+              label="z"
+              incrementor="slider"
+              :value="destinationMarker.z"
+              step="0.1"
+              @input="handleUpdateZ($event)"
+            />
+          </div>
+          <v-button
+            class="pt-1"
+            label="Move"
+            variant="success"
+            icon="play-circle-filled"
+            :disabled="moveClick ? 'false' : 'true'"
+            @click="executeMoveOnMap()"
+          />
+          <v-switch
+            class="pt-2"
+            label="Show Axes"
```

Comment:
This wasn't part of the design. Where was this addition discussed?

Reply:
yep, this was a design-consulted decision

Reply:
thanks!